backend/src/main/java/com/benine/backend/http/RequestHandler.java

```diff
@@ -45,6 +45,22 @@ public Attributes parseURI(String uri) throws MalformedURIException {
   }
 
   /**
+   * Formats the response message for a success or failure.
+   * @param exchange the HttpExchange.
+   * @param correct boolean that is true if the exchange is successful. False otherwise.
+   */
+  public void responseMessage(HttpExchange exchange, boolean correct) {
```

Comment:
I agree with Dorian about the method splitting, it spreads the behavior more nicely.